packages/database/src/aggregated-liveness/repository.test.ts

```diff
@@ -22,7 +19,7 @@ describeDatabase(AggregatedLivenessRepository.name, (database) => {
       min: 10,
       avg: 10,
       max: 10,
-      updatedAt: START.add(-1, 'hours'),
+      timestamp: START.add(-1, 'hours'),
```

Comment:
The correct name of this field should be `updatedAt`. Let's change it `AggregatedLivenessRecord`. I think it was overlooked before. Underlying db filed is called `updated_at` so it will be consistent.